### PR TITLE
Added POS to `Hardware categories` page

### PR DESCRIPTION
**Pull request by Bosch-0**
Closes https://github.com/BitcoinDesign/Guide/issues/584  

Adds a point of sale (POS) section to the hardware categories page. 

📲[Check the preview](https://deploy-preview-773--sad-borg-390916.netlify.app/guide/getting-started/hardware/#point-of-sale)💻

### CODE REVIEW

File: guide/getting-started/hardware.md [hunk 1]
```diff
@@ -99,11 +101,11 @@ It is quite common to have dedicated hardware to run bitcoin [node software]({{
...
+Heavy users of the Lightning Network, such as routing node operators, often run Lightning nodes on dedicated hardware. Lightning nodes require 24/7 uptime, which is much easier to achieve with dedicated hardware. Lightning nodes also require the private keys to be stored on the same device. Dedicated hardware reduces the attack surface and makes it easier to secure private keys.
 
 [Plug-and-play hardware nodes](https://samouraiwallet.com/nodl) exist and make it simple to set up a dedicated hardware node. The most common way people run dedicated hardware nodes is installing a node OS, like [Umbrel](https://getumbrel.com/) or [MyNode](https://mynodebtc.com/), on a hardware device.
 
-See [nodes]({{ '/guide/glossary/node/' | relative_url }}) for more details.
+See our [nodes]({{ '/guide/glossary/node/' | relative_url }}) for more details on how these work.
```
Comment (Bosch-0) [inline guide/getting-started/hardware.md:108]:
```suggestion
See our [nodes]({{ '/guide/glossary/node/' | relative_url }}) page for more details on how they work.
```

File: guide/getting-started/hardware.md [hunk 2]
```diff
@@ -77,7 +79,7 @@ You may already be familiar with physical security keys from your bank or work.
    height = 400
 %}
 
-Bitcoin hardware wallets, also known as external signers, act like bitcoin-centric security keys. They isolate the recovery phrase and private keys from the internet and other devices.
+Bitcoin hardware wallets act like bitcoin-centric security keys. They isolate the recovery phrase, private keys and other sensitive data from the internet and other devices.
```
Comment (sbddesign) [inline guide/getting-started/hardware.md:82]:
```suggestion
Bitcoin hardware wallets act like bitcoin-centric security keys. They isolate the recovery phrase, private keys, and other sensitive data from the internet and other devices.
```

Comment (sbddesign) [inline guide/getting-started/hardware.md:82]: What's an example of "other sensitive data" for a hardware wallet?

File: guide/getting-started/hardware.md [hunk 3]
```diff
@@ -54,15 +54,17 @@ General purpose hardware, like personal computers and smartphones, work great fo
...
 
-A simpler option is writing your recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
+A simpler option is writing a recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
 
 Some applications, such as [Muun](https://blog.muun.com/why-not-just-a-mnemonic/) have users print out encrypted PDF backups that contain all the required recovery data, not just the recovery phrase.
 
 See [bitcoin backups]({{ '/guide/private-key-management/backups/' | relative_url }}) for more details.
 
-## Hardware wallets
+## External signers
```
Comment (sbddesign) [inline guide/getting-started/hardware.md:65]:
```suggestion
## Hardware wallets
```
I noticed it says "also known as external signers" immediately below this, so assuming we should change the title back to hardware wallets.

File: guide/getting-started/hardware.md [hunk 4]
```diff
@@ -132,7 +134,7 @@ Bitcoin Automated Teller Machines (ATM) are a convenient way to buy or sell bitc
    retina = "/assets/images/guide/getting-started/hardware/atm@2x.jpg"
    mobile = "/assets/images/guide/getting-started/hardware/atm-mobile.jpg"
    mobileRetina = "/assets/images/guide/getting-started/hardware/atm-mobile@2x.jpg"
-   alt-text = "Examples of hardware wallets"
+   alt-text = "Examples of hardware wallets."
```
Comment (sbddesign) [inline guide/getting-started/hardware.md:137]:
```suggestion
   alt-text = "Example of a bitcoin ATM."
```

File: guide/getting-started/hardware.md [hunk 5]
```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
...
+   alt-text = "Examples of point of sale hardware devices."
+   width = 800
+   height = 400
+%}
+
+Dedicated bitcoin POS hardware is used to request and receive bitcoin payments in-person. These act very similar to traditional POS systems but present to payees a bitcoin on-chain, lightning, or [unified](https://bitcoinqr.dev/) payment request. Payments can also be received over NFC.
+
+As a user needs to be online to receive a lightning payment, POS terminals can act as a always on lightning node allowing lightning payments to be received easily.
+
+Accepting lightning payments with POS hardware can also be offline using a standard like [LNURL](https://github.com/fiatjaf/lnurl-rfc) to easily receive payments. An example of this is [LNURLPoS](https://github.com/arcbtc/LNURLPoS)
```
Comment (sbddesign) [inline guide/getting-started/hardware.md:168]: I agree with everything being said here, but I think this and the above paragraph make it seem like there are two ways to make a bitcoin POS:
- Always on lightning node embedded in POS terminal
- Completely offline LNURLPoS

In my experience, these are the least common bitcoin POS systems. The most common bitcoin POS system is:
- POS terminal which connects remotely to a lightning node with API calls (e.g. using Chivo, Ibex, or OpenNode, or even accessing one's BTCPayServer from an iPad).

File: guide/getting-started/hardware.md [hunk 6]
```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
...
+
+{% include picture.html
+   image = "/assets/images/guide/getting-started/hardware/point-of-sale.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/point-of-sale@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile@2x.jpg"
+   alt-text = "Examples of point of sale hardware devices."
+   width = 800
+   height = 400
+%}
```
Comment (sbddesign) [inline guide/getting-started/hardware.md:162]: Would it be a good idea to include a tablet in this image as well? Something like one of the Square POS tablets mounted on the swivel? While the tiny little brick device is definitely a thing, many businesses accepting bitcoin are doing it through a tablet based POS and I only see that trend continuing.

I propose this in the same spirit as the hardware wallets image which shows devices of all shapes and sizes.

**Review by sbddesign — CHANGES_REQUESTED** [review]
Good stuff. I have some discussion points.

File: guide/getting-started/hardware.md (hunk 2 above)
Comment (Bosch-0) [reply guide/getting-started/hardware.md]: Descriptors / xpub are two that come to mind.

File: guide/getting-started/hardware.md (hunk 3 above)
Comment (Bosch-0) [reply guide/getting-started/hardware.md:65]: I thought I did change it back, my bad thanks for that !

File: guide/getting-started/hardware.md (hunk 6 above)
Comment (Bosch-0) [reply guide/getting-started/hardware.md:162]: Yeah I can add that, good suggestion!

File: guide/getting-started/hardware.md (hunk 2 above)
Comment (Bosch-0) [reply guide/getting-started/hardware.md]: xpub is derived from the PK so it isn't really stored, descriptors though are stored though - I'm pretty sure this is the case with Coldcard (one of the only HWW fully supporting descriptors atm).

File: guide/getting-started/hardware.md (hunk 5 above)
Comment (Bosch-0) [reply guide/getting-started/hardware.md:168]: Made changes to reflect these points, though your last point would probably be more suitable (and already is very minimally) in the software categories page.

File: guide/getting-started/hardware.md [hunk 7]
```diff
@@ -54,16 +54,18 @@ General purpose hardware, like personal computers and smartphones, work great fo
...
-A simpler option is writing your recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
+A simpler option is writing a recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
 
 Some applications, such as [Muun](https://blog.muun.com/why-not-just-a-mnemonic/) have users print out encrypted PDF backups that contain all the required recovery data, not just the recovery phrase.
 
 See [bitcoin backups]({{ '/guide/private-key-management/backups/' | relative_url }}) for more details.
 
 ## Hardware wallets
 
+_Also known as external signers_
```
Comment (GBKS) [inline guide/getting-started/hardware.md:67]: This looks weird as a stand-alone sentence on the preview page. Adding a period might help.

File: guide/getting-started/hardware.md [hunk 8]
```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
...
+   image = "/assets/images/guide/getting-started/hardware/point-of-sale.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/point-of-sale@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile@2x.jpg"
+   alt-text = "Examples of point of sale hardware devices."
+   width = 800
+   height = 400
+%}
+
+Dedicated bitcoin POS hardware is used to request and receive bitcoin payments in-person. These act very similar to traditional POS systems but present to payees a bitcoin on-chain, lightning, or [unified](https://bitcoinqr.dev/) payment request. Payment requests can also be shared and paid over NFC.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:164]: Not sure how important it is to point out the different types of payment requests here...

For NFC, maybe add "if the POS hardware and user devices support it"?

File: guide/getting-started/hardware.md [hunk 9]
```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
...
+
+When accepting lightning payments, these POS terminals often connect to a remote always on lightning node. An example of is a merchant using a dedicated iPad to connect to [BTCPay Server](https://btcpayserver.org/) instance.
```
Comment (GBKS) [inline guide/getting-started/hardware.md:166]:
```suggestion
When accepting lightning payments, these POS terminals often connect to a remote, always-on lightning node. For example, a merchant may use a dedicated iPad to connect to a [BTCPay Server](https://btcpayserver.org/) instance.
```

File: guide/getting-started/hardware.md [hunk 10]
```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
...
+
+Accepting lightning payments with POS hardware can also be fully offline using a standard like [LNURL](https://github.com/fiatjaf/lnurl-rfc) to easily receive payments. An example of this is [LNURLPoS](https://github.com/arcbtc/LNURLPoS)
```
Comment (GBKS) [inline guide/getting-started/hardware.md:168]:
```suggestion
Accepting lightning payments with POS hardware can also be fully offline, using a standard like [LNURL](https://github.com/fiatjaf/lnurl-rfc) to easily receive payments. An example of this is [LNURLPoS](https://github.com/arcbtc/LNURLPoS).
```
LNURLPoS is a horrible name for a product, hope that will be changed at some point.

**Comment by GBKS** [comment]
Just left a few comments. In the image, you could add a simplified representation of an amount input (big number, keypad button outlines below) to make it a bit more obvious what the iPad is used for.

File: guide/getting-started/hardware.md (hunk 10 above)
Comment (sbddesign) [reply guide/getting-started/hardware.md:168]: Agreed on the LNURLPoS thing, but I'd argue it's the name of a technology and not a product. I think Ben is putting the tech out there as FOSS so other people can turn it into an actual product. So my thinking is that any individual or company who picks up his tech will come up with their own unique name for the product.

File: guide/getting-started/hardware.md (hunk 7 above)
Comment (sbddesign) [reply guide/getting-started/hardware.md:67]: Without punctuation, it functions like a subheading. Adding a period would turn it into an incomplete sentence, so we shouldn't do that, @GBKS. 

@Bosch-0 I'd recommend doing one of these:

- Leaving as is
- Merging it with one of the sentences, e.g. "Bitcoin hardware wallets, also known as _signers_, act like bitcoin-centric security keys."
- Add it to a [tip box](https://bitcoin.design/guide/contribute/formatting/#tips).

**Review by sbddesign — APPROVED** [review]
LGTM!
I left a comment regarding how to treat the dangling "external signers" sentence. I'll let you and Christoph discuss, and I will review again if needed.

File: guide/getting-started/hardware.md (hunk 7 above)
Comment (Bosch-0) [reply guide/getting-started/hardware.md:67]: I'll go with point 2

**Review by GBKS — APPROVED** [review]
ACK. Good addition.

**Pull request merged**